### PR TITLE
Add codecov badge to readme

### LINKED ISSUE #33: Add Travis CI build badge to README.md
Now that we have Travis CI set up here: https://travis-ci.com/paypal/NNAnalytics
It would be cool to get the badge posted on our README, like so:
```
[![Build Status](https://travis-ci.com/paypal/NNAnalytics.svg?branch=master)](https://travis-ci.com/paypal/NNAnalytics)
```

### PR DESCRIPTION
**Pull request by r39132**
Make sure you have checked all steps below.

### GitHub Issue
Fixes #33 

### Checklist:

- [x] This pull request updates the documentation
- [ ] This pull request changes the code
- [ ] Title of the PR is of format (example) : `[#25] Add Pull Request Template`, where `[#25] is the GitHub issue number`

### What is the purpose of this pull request?

Show code-cov badge on the README

### Commit Guidelines
- [x] My commits all reference GH issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    2. Subject is limited to 50 characters
    3. Subject does not end with a period
    4. Subject uses the imperative mood ("add", not "adding")
    5. Body wraps at 72 characters
    6. Body explains "what" and "why", not "how"

### CODE REVIEW

**Comment by zero45** [comment]
Seems this branch is on top of `minicluster` branch. Did you intend to focus on `minicluster` first and then come back to this?

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+[![Build Status](https://travis-ci.org/paypal/NNAnalytics.svg?branch=master)](https://travis-ci.com/paypal/NNAnalytics)
```
Comment (zero45) [inline README.md]: I think `https://travis-ci.org/paypal/NNAnalytics.svg?branch=master` -> `https://travis-ci.com/paypal/NNAnalytics.svg?branch=master`, right?

Comment (jhsenjaliya) [reply README.md]: needs travis-ci.com in both places.

**Comment by jhsenjaliya** [comment]
+1, link works with correct status.

**Pull request merged**